tests/modules/reply.c

```diff
@@ -47,6 +47,12 @@ int rw_longdouble(RedisModuleCtx *ctx, RedisModuleString **argv, int argc) {
     return RedisModule_ReplyWithLongDouble(ctx, longdbl);
 }
 
+int rw_bignumber(RedisModuleCtx *ctx, RedisModuleString **argv, int argc) {
+    if (argc != 1) return RedisModule_WrongArity(ctx);
+
+    return RedisModule_ReplyWithBigNumber(ctx, "12345678901234567890");
```

Comment:
Why does this compile success(The last parameter is missing)?

Reply:
1. `--single unit/moduleapi/reply` is missing in `runtest-moduleapi`
2. `reply.so` is missing in `test/module/Makefile`

Reply:
good catch.. it also seems that the existing tests there are broken.
this was overlooked when #8521 was merged, luckily none of that is released yet, so even if there are bugs we can safely fix them.
@sjpotter please fix this as part of this PR.